packages/core/upload/admin/src/components/TableList/TableRows.js

```diff
@@ -22,8 +33,16 @@ export const TableRows = ({ onEditAsset, onEditFolder, onSelectOne, rows, select
         const isSelected = !!selected.find((currentRow) => currentRow.id === id);
 
         return (
-          <Tr key={id}>
-            <Td>
+          <Tr
+            key={id}
+            {...onRowClick({
```

Comment:
```suggestion
            {...onRowClick({ fn: handleRowClickFn(element, elementType) })
```

Reply:
I can't figure out how to suggest multiple lines so you won't actually be able to just commit this, but in essence it makes the inline code easier to read :)